### DISCUSSION TITLE
Important questions about xudp and udp443 in reality vision

### DISCUSSION THREAD

**Discussion by MrVb0** [XTLS/Xray-core#2018] [Q&A]
HELLO
Greetings to xray developers and other loved ones who are working towards free internet access
Thank you very much for helping us and we appreciate your efforts
It has been a while since the xudp and udp443 features were added to the core, and according to the explanations I have read, I realized that they have a significant effect in preventing filtering.
But I didn't find detailed instructions, examples and explanations about the type of best configuration
- Is using udp443 better than not using it? Why ?
- Is using xudp better than not using it? Why ?
- What is the best configuration example for Reality? (If using udp443 and xudp helps, include it as well)

**Comment by us254** [comment]
https://github.com/XTLS/Xray-core/issues/1963#issuecomment-1512532299

https://t.me/projectXtls/68
